src/test/kotlin/nl/info/client/brp/BrpClientServiceTest.kt

```diff
@@ -5,19 +5,28 @@
 
 package nl.info.client.brp
 
+import io.kotest.assertions.throwables.shouldThrow
 import io.kotest.core.spec.style.BehaviorSpec
 import io.kotest.matchers.shouldBe
 import io.mockk.checkUnnecessaryStub
 import io.mockk.every
 import io.mockk.mockk
+import nl.info.client.brp.exception.BrpInvalidPurposeException
 import nl.info.client.brp.model.createPersoon
 import nl.info.client.brp.model.createRaadpleegMetBurgerservicenummer
 import nl.info.client.brp.model.createRaadpleegMetBurgerservicenummerResponse
+import java.util.Optional
 
+const val PURPOSE_SEARCH = "customPurpose"
```

Comment:
```suggestion
const val QUERY_PERSONEN_PURPOSE = "testQueryPurpose"
```

Reply:
Fine for now but for future PRs: our convention for test data is mostly to use the word fake. So e.g. `fakeQueryPurpose` in this case.